src/views/PeopleSearch/components/SearchFieldList/index.tsx

```diff
@@ -369,11 +371,11 @@ const SearchFieldList = ({ onSearch }: Props) => {
         <Grid container alignItems="center">
           <Accordion style={{ flexGrow: 1 }} elevation={3}>
             <AccordionSummary
-              expandIcon={<ExpandMore />}
+              expandIcon={<ExpandMore className={styles.people_expand_icon} />}
```

Comment:
This class doesn't exist in the imported stylesheet.

Reply:
See above comment - fixed